addons/project/populate/project.py

```diff
@@ -57,15 +57,37 @@ def get_stage_ids(random, **kwargs):
 class ProjectTask(models.Model):
     _inherit = "project.task"
     _populate_sizes = {"small": 500, "medium": 5000, "large": 50000}
-    _populate_dependencies = ["project.project"]
+    _populate_dependencies = ["project.project", "res.partner", "res.users"]
 
     def _populate_factories(self):
         project_ids = self.env.registry.populated_models["project.project"]
         stage_ids = self.env.registry.populated_models["project.task.type"]
+        user_ids = self.env.registry.populated_models['res.users']
+        partner_per_company = {
```

Comment:
```suggestion
        partner_ids_per_company_id = {
```
to be precise :angel: